packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -70,6 +71,8 @@ export const direction = create<Direction>("direction").withDefault(Direction.lt
 /** @public */
 export const disabledOpacity = create<number>("disabled-opacity").withDefault(0.3);
 /** @public */
+export const layerCornerRadius = create<number>("layer-corner-radius").withDefault(6);
```

Comment:
Please follow https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/design-tokens.ts as much as possible in terms of sequence, commenting, helper functions, etc. We may have new design tokens, which can fall in line alphabetically within "foreground", "fill", and "stroke" sections.

Reply:
rearranged and formatted